tests/conftest.py

```diff
@@ -37,7 +37,7 @@
 
     from _pytest.python import Metafunc
 
-GH_MATRIX_LENGTH = 37
+GH_MATRIX_LENGTH = 34
```

Comment:
@alisonlhart This is based on my understanding of what is happening, need your +1 here